client/app/services/auth-guard-login.service.ts

```diff
@@ -8,7 +8,11 @@ export class AuthGuardLogin implements CanActivate {
   constructor(public auth: AuthService, private router: Router) {}
 
   canActivate() {
-    return this.auth.loggedIn;
+    if (this.auth.loggedIn()) {
```

Comment:
At the time of writing while referring to [angular2-jwt documentation](https://github.com/auth0/angular2-jwt#checking-authentication-to-hideshow-elements-and-handle-routing) I considered that but just left the code ready for a redirect like in the usage example.  Thought that I'd let Davide see the pull request if helpful or requires changes.